src/Microsoft.Sbom.Common/Microsoft.Sbom.Common.csproj

```diff
@@ -10,7 +10,6 @@
 
   <ItemGroup>
     <PackageReference Include="Serilog.Sinks.Console" />
-    <PackageReference Include="System.IO.FileSystem.AccessControl" />
```

Comment:
What is the reason for the change in this file?

Reply:
Same as previous